packages/solid/test/signals.spec.ts

```diff
@@ -160,9 +160,9 @@ describe("Update signals", () => {
   });
   test("Set signal returns argument", () => {
     const [_, setValue] = createSignal<number>();
-    const res1: undefined = setValue(undefined);
+    const res1: number | undefined = setValue(undefined);
     expect(res1).toBe(undefined);
-    const res2: number = setValue(12);
+    const res2: number | undefined = setValue(12);
```

Comment:
Oh, um, this change. Hmmmmm. Maybe it wasn't a bug fix like my commit message thought.

Reply:
converting back to draft

Reply:
fixed